docs/install.rst

```diff
@@ -12,6 +12,15 @@ PostgreSQL 11.
 Ubuntu or Debian
 ----------------
 
+Quick install
+~~~~~~~~~~~~~
+
+The following installation method downloads a bash script that automates
+several steps. The full script is available for review at our `package cloud
+installation instructions page`__ url.
```

Comment:
may drop `url` word

Reply:
more like
package cloud installation instructions __ page.

Reply:
Fixed that way, thanks!